rules/manager.go

```diff
@@ -87,6 +87,10 @@ func DefaultEvalIterationFunc(ctx context.Context, g *Group, evalTimestamp time.
 	g.setLastEvalTimestamp(evalTimestamp)
 }
 
+func DefaultEvaluateRuleGroupFunc(ctx context.Context, g *Group, evalTimestamp time.Time, doneCh <-chan struct{}) bool {
```

Comment:
Can we add comment?  Also name is wrong, this should be perhaps:
```suggestion
func AlwaysTrueShouldEvaluateFunc(ctx context.Context, g *Group, evalTimestamp time.Time, doneCh <-chan struct{}) bool {
```
WDYT?

Reply:
Like this! Will make the changes soon